docs/src/examples/components/Indicator/Types/IndicatorExampleDirection.shorthand.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { Indicator } from '@stardust-ui/react'
+
+const IndicatorExampleDirection = () => (
+  <div>
```

Comment:
May be `Fragment` API?

```suggestion
  <>
```